### PR TITLE
Fixed a little something for compression checks

### PR DESCRIPTION
**Pull request by Krzmbrzl**
### When merged this pull request will:

1. Fix a function header
2. Fix logging during compression test

### CODE REVIEW

File: addons/CLib/Core/Compression/fn_checkAllFunctionCompression.sqf [hunk 1]
```diff
@@ -18,7 +18,7 @@
     private _fncName = _x;
     private _originalFunction = (parsingNamespace getVariable _fncName) call CFUNC(codeToString);
 
-    if (_originalFunction call CFUNC(checkCompression)) then {
+    if !(_originalFunction call CFUNC(checkCompression)) then {
```
Comment (jokoho48) [inline addons/CLib/Core/Compression/fn_checkAllFunctionCompression.sqf]: instead of using ! here why not just switch the contents around?

Comment (Krzmbrzl) [reply addons/CLib/Core/Compression/fn_checkAllFunctionCompression.sqf]: No real reason... It's just what came into my mind first.
If you want, I can change it though

Comment (jokoho48) [reply addons/CLib/Core/Compression/fn_checkAllFunctionCompression.sqf]: its cleaner to read

**Comment by jokoho48** [comment]
can get removed was left over from a time where we had 3 compression styles

**Comment by Krzmbrzl** [comment]
Done

**Pull request merged**